app/helpers/application_helper.rb

```diff
@@ -20,6 +19,19 @@ def self.icon_specified?(icon_name)
     ICON_MAP[ICON_PACK][icon_name].present?
   end
 
+  def pick_icons(icon_pack, icons)
+    ICON_MAP[icon_pack].slice(*icons)
+  end
+
+  def icon_class_tag(icon_class, additional_classes = [])
+    classes_string = [icon_class].concat(additional_classes).join(" ")
+    "<i class=\"#{classes_string}\"></i>".html_safe
+  end
+
+  def icon_map_tag(icon_map, icon_name, additional_classes = [])
+    icon_class_tag(icon_map[icon_name], additional_classes)
+  end
+
```

Comment:
Can all the icon stuff be pulled to a separate module that is mixed in (included) to application helper?